### DISCUSSION TITLE
Font size

### DISCUSSION THREAD

**Discussion by LalFX** [gnikit/tkinter-tooltip#45] [Q&A] [answered]
Thanks for the implementation. 

I could not find a way to increase the font size anywhere. Is there a simple way to do this that I missed?

**Comment by gnikit** [comment, marked as answer]
You can pass your font like you would normally in a tkinter widget in the `**message_kwards` of the tooltip

```python
 ToolTip(label, msg=item, font=("Arial", 12))
```
